benchmarks/nemo/nemolite2d/psykal/psyclone_scripts/acc_transform.py

```diff
@@ -14,7 +15,7 @@ def trans(psy):
     enter_data_trans = tinfo.get_trans_name('ACCEnterDataTrans')
     routine_trans = tinfo.get_trans_name('ACCRoutineTrans')
     glo2arg_trans = tinfo.get_trans_name('KernelImportsToArguments')
-    inline_trans = tinfo.get_trans_name('KernelModuleInline')
```

Comment:
`nemo/nemolite2d/psykal/psyclone_scripts/omp_transform.py` has the same issue, can you update it there as well.

Reply:
Oh yes, fixed. That must mean that the 'gen' target doesn't build the OMP version and perhaps it should?

Reply:
I've extended it so that it does. I thought I could do the same for the ACC kernels target but that fails. I'll open a separate issue for that.

Reply:
For reference, the issue is a very old one - #12.